alf/networks/q_networks.py

```diff
@@ -20,14 +20,14 @@
 import torch.nn as nn
 
 import alf.layers as layers
-from alf.networks import EncodingNetwork, LSTMEncodingNetwork
-from alf.networks import PreprocessorNetwork
+from alf.networks import EncodingNetwork, LSTMEncodingNetwork, ParallelEncodingNetwork
+from alf.networks import Network
 from alf.tensor_specs import TensorSpec, BoundedTensorSpec
 import alf.utils.math_ops as math_ops
 
 
 @gin.configurable
-class QNetwork(PreprocessorNetwork):
```

Comment:
We need to derive from ```PreprocessorNetwork``` instead of ```Network```?

Reply:
preprocessing is handled by `self._encoding_net`; so no need to derive from that